english/explore-outside/talk-child/talk-child.md

```diff
@@ -3,4 +3,6 @@ The child looks towards you and you see red eyes garing at you.
 
 Do you:
 
-[Decide to head home because red eyes are creepy for reals.](../journey-to-flaming-tower/home/home.md)
\ No newline at end of file
+[Decide to head home because red eyes are creepy for reals.](../journey-to-flaming-tower/home/home.md)
+
+[Take your cellphone out, call Tank and ask for the nearest exit out of the Matrix.](../journey-to-flaming-tower/exit/exit.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.